src/Utils/Strings.php

```diff
@@ -332,10 +332,10 @@ public static function compare($left, $right, $len = NULL)
 	 * @param  string|array
 	 * @return string
 	 */
-	public static function findPrefix($strings)
+	public static function findPrefix(...$strings)
 	{
-		if (!is_array($strings)) {
-			$strings = func_get_args();
+		if (is_array($strings[0])) {
+			$strings = $strings[0];
```

Comment:
That is true, also the name implies one string argument, not an array. In PHP 7 we should pick one and stick with it (`array $strings` vs `string ...$strings`).
This is a common scenario through Nette though, e.g. nette/finder#4 is similar.
Either way this is a good thing to think about - do we want to prefer variadics & unpacking or untyped arrays?